V/viterbi/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libviterbi"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/zsoerenm/viterbi.git", "5a9411d48708adcb84ff2250fd3fa7b1a2c131fe")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd viterbi
+mkdir $libdir
+make
+cp libviterbi.so $libdir/
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/viterbi
make
install -Dvm libviterbi.so "${libdir}/libviterbi.${dlext}"
```
Although it would be nicer if the `.so` extension wasn't hardcoded, because the library for macOS and Windows then has the wrong SONAME.